docs/introduction.rst

```diff
@@ -12,15 +12,27 @@ Dependencies
 Data levels
 ===========
 
-* R0
-* R1
-* DL1a
-* DL1b
-* DL2
-* DL3
++------+---------------------------------------------------------------------------+-------------+
+| Level| Description                                                               | File Format |
++======+===========================================================================+=============+
+| R0   | Uncalibrated RAW waveforms from the camera                                | ZFITS       |
++------+---------------------------------------------------------------------------+-------------+
+| R1   | Calibrated waveforms from the camera                                      |             |
++------+---------------------------------------------------------------------------+-------------+
+| DL1a | Integrated charge and peak position of the waveform                       | HDF5        |
++------+---------------------------------------------------------------------------+-------------+
+| DL1b | Image parameters (width, length, intensity, etc.)                         | HDF5        |
++------+---------------------------------------------------------------------------+-------------+
+| DL2  | Event parameters (energy, direction, time, etc.)                          | HDF5        |
++------+---------------------------------------------------------------------------+-------------+
+| DL3  | Lists of reconstructed events after event selection                       | FITS        |
+|      | with corresponding IRFs (AEFF, EDISP, PSF, etc.)                           |             |
```

Comment:
1 small shift in the column border at the end

Reply:
Thx, fixed.